metadata.rb

```diff
@@ -5,7 +5,7 @@
 description 'Installs/Configures grafana'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 
-version '0.3.0'
+version '0.4.0'
```

Comment:
don't touch metadata here in case other versions get pushed out. When we release we'll do this. for the changelog just add changes to the top of the file with "UNRELEASED" as the version string.